### ISSUE TITLE
JSONStringify with undefined cause TypeError

### ISSUE THREAD

**Issue by neko-para** [Ivan-Korolenko/json-with-bigint#9]
Though `undefined` is acceptable according to dts file, `undefined` itself cannot be passed directly into `JSONStringify` as `JSON.stringify(undefined)` will result `undefined`, and then calling `replace` on it will cause a `TypeError`. Consider use `Exclude` to exclude it.

![image](https://github.com/user-attachments/assets/431ea540-79cc-4e17-b0ec-0e6d61be353f)

---

[ref](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify)

![image](https://github.com/user-attachments/assets/b60d7b18-7664-49df-9fc1-7a7d547792ed)

---

Luckily `null` is ok as `JSON.stringify(null)` results `'null'`.

**Comment by neko-para** [comment]
Adding an extra declaration is also a possible choice. (And add `?.` for `replace` inside)

```typescript
declare function JSONStringify(data: undefined, space?: string | number): undefined
///
preliminaryJSON?.replace(bigInts, "$1$2$3")
```

**Comment by Ivan-Korolenko** [comment]
Hi @neko-para,

Thank you for bringing my attention to this. **It's fixed in version 2.3.3** and already available on npm.

Thank you for your interest in the `json-with-bigint` library. Feel free to create a new issue or a PR if you encounter any problems with the library in the future or if you would like to propose some feature or a change.
